tests/test_blocks.py

```diff
@@ -397,3 +398,55 @@ def test_renders(self):
             '<a href="https://omni-digital.co.uk">Omni Digital</a>',
             button_block.render(value)
         )
+
+
+class TestFlowBlock(TestCase):
+    block = internal_blocks.FlowBlock()
+
+    def test_meta_title(self):
+        """Ensure FlowBlock.meta_title is the expected block type."""
+        child_block = self.block.child_blocks['meta_title']
+
+        self.assertIsInstance(child_block, blocks.CharBlock)
+        self.assertTrue(child_block.required)
+
+    def test_title(self):
+        """Ensure FlowBlock.title is the expected block type."""
+        child_block = self.block.child_blocks['title']
+
+        self.assertIsInstance(child_block, blocks.CharBlock)
+        self.assertFalse(child_block.required)
+
+    def test_body(self):
+        """Ensure FlowBlock.body is the expected block type."""
+        child_block = self.block.child_blocks['body']
+
+        self.assertIsInstance(child_block, blocks.RichTextBlock)
+        self.assertFalse(child_block.required)
+
+    def test_image(self):
+        """Ensure FlowBlock.image is the expected block type."""
+        child_block = self.block.child_blocks['image']
+
+        self.assertIsInstance(child_block, ImageChooserBlock)
+        self.assertFalse(child_block.required)
+
+    def test_link(self):
+        """Ensure FlowBlock.link is the expected block type."""
+        child_block = self.block.child_blocks['link']
+
+        self.assertIsInstance(child_block, internal_blocks.LinkBlock)
+        self.assertFalse(child_block.required)
+
+    def test_renders(self):
+        """Ensure FlowBlock renders as expected."""
+        value = self.block.to_python({
+            'meta_title': 'Meta Title',
+            'title': 'Main Title',
+            'body': 'This is the body.',
```

Comment:
would it be possible to add the child link blocks into this test or as a separate render test?

Reply:
Done